modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,156 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+:_content-type: PROCEDURE
+[id='replacing-a-bare-metal-control-plane-node_{context}']
+= Replacing a bare metal control plane node
+
+Use the following procedure to replace an installer-provisioned {product-title} cluster.
+
+[IMPORTANT]
+====
+When the replacement BareMetalHost is created from a new physical machine or an incomplete prior YAML file configuration, the `externallyProvisioned:` field in the configuration file must be set to `false` to allow the cluster to provision the host. 
+
+Existing control plane BareMetalHosts may have the externallyProvisioned flag set to true if they were provisioned by the OpenShift Installer.
```

Comment:
I imagine there's probably some `{product-title}`-type macros that are supposed to be used here, but I don't know the details.